src/frontend/auth0_templates/invite_new_user_email_template_prod.html

```diff
@@ -44,20 +43,16 @@
             style='-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;mso-table-lspace: 0pt;mso-table-rspace: 0pt;margin: 0;padding: 20px;font-family: "ProximaNova", sans-serif;height: 100% !important;'
           >
             <div class="main">
-              
-							<div style='padding: 20px;font-size:25px;font-weight:700;font-size:22px;line-height:25.3px;font-family:helvetica;color:black'> You've been invited to join {% if organization.name %}{{organization.name}}'s {% else %} a {% endif %} group on CZ GEN EPI</div>
-            
-              <div style='text-align:left;margin-top:20px;color:black'>Click on the link below to accept this invite and get started. If you do not already have an account with CZ GEN EPI, you will need to make one to get started. This invitation will expire after 14 days. </div>
-
-              <button style='margin-top: 40px;background-color:#511CC1;border-radius:40px;border:none;padding: 10px 30px;font-size:13px;line-height:18px;font-weight:400;'>
-                <span style='color:white;text-decoration:none'>
-                  <a style='color:white;text-decoration:none' href={{ url }} >Accept Invitation</a>
+              <div style='padding: 20px;font-size:25px;font-weight:700;font-size:22px;line-height:25.3px;font-family:helvetica;color:black;'> You've been invited to join {% if organization.name %}{{organization.name}}'s {% else %} a {% endif %} group on CZ GEN EPI</div>
```

Comment:
woops this should be display_name
```suggestion
              <div style='padding: 20px;font-size:25px;font-weight:700;font-size:22px;line-height:25.3px;font-family:helvetica;color:black;'> You've been invited to join {% if organization.display_name %}{{organization.display_name}}'s {% else %} a {% endif %} group on CZ GEN EPI</div>
```